gnomad_qc/v4/assessment/summary_stats.py

```diff
@@ -35,10 +35,11 @@ def main(args):
     test = args.test
     overwrite = args.overwrite
     data_type = args.data_type
+    public = args.get_public_sites
```

Comment:
since this is an argument, I don't think public needs to be True by default in the resources (but I also don't see a reason why we shouldn't read from the public bucket rather than our own buckets by default) 😆